etl/multidim.py

```diff
@@ -875,3 +791,26 @@ def _check_intersection_iters(
         raise ValueError(
             f"Unexpected items: {', '.join([f'`{d}`' for d in items_unexpected])}. Please review `{key_name}`!"
         )
+
+
+def group_views(views: dict[str, Any], by: list[str]) -> list[dict[str, Any]]:
```

Comment:
I have some questions on the "assumptions" that this function does on the input. E.g.

- what if a view contains multiple indicators already?
- what about other dimensions like `x`?

Maybe good to clarify when this would work and when not?

Reply:
Good points. I didn't spend much time polishing it as I thought it's pretty rare case, but since you said you might use it elsewhere, I'll find some time to improve it.